docs/guide/content_rendering.md

```diff
@@ -1384,30 +1290,30 @@ integer **number\_of\_bytes**|ez\_file\_size( integer **number\_of\_decimal** )
 | Argument name | Type                                                   | Description                               |
 |---------------|--------------------------------------------------------|-------------------------------------------|
 | `field`       | `eZ\Publish\API\Repository\Values\Content\Field`       | The image Field                           |
-| `versionInfo` | `eZ\Publish\API\Repository\Values\Content\VersionInfo` | The VersionInfo that the field belongs to |
-| `variantName` | `string`                                               | Name of the image alias to be used        |
+| `versionInfo` | `eZ\Publish\API\Repository\Values\Content\VersionInfo` | The VersionInfo that the Field belongs to |
+| `variantName` | `string`                                               | Name of the image variation to be used        |
 
-#### ez\_is\_field\_empty
+See [images](images.md) for more information about image variations.
+
+#### `ez_is_field_empty`
 
 ##### Description
 
-`ez_is_field_empty()` is a Twig helper which checks if a Content item's field value is considered empty in the current language.
+`ez_is_field_empty()` is a Twig helper which checks if a Content item's Field value is considered empty in the current language.
 
 It returns a boolean value (`true` or `false`).
 
-!!! note
-
-    If the current language cannot be found as a translation for content, the main language will be used. This behavior is identical when forcing a language.
+If the Content item does not have a translation in the current language, the main language will be used. This behavior is identical when forcing a language using **forcedLanguage**.
 
 ##### Prototype and Arguments
 
-`ez_is_field_empty( eZ\Publish\API\Repository\Values\Content\Content content, eZ\Publish\API\Repository\Values\Content\Field|string **fieldDefIdentifier**\[, string **forcedLanguage**\] **)**`
+`ez_is_field_empty( eZ\Publish\API\Repository\Values\Content\Content content, eZ\Publish\API\Repository\Values\Content\Field|string fieldDefIdentifier[, string forcedLanguage] )`
 
 | Argument name | Type | Description |
 |---------------|------|-------------|
-| `content` | `eZ\Publish\API\Repository\Values\Content\Content` | Content object the displayable field belongs to. |
-| `fieldDefIdentifier` | `eZ\Publish\API\Repository\Values\Content\Field or string` | The field we want to check or its identifier. |
-| `forcedLanguage` | `string` | Locale we want the content name translation in (e.g. "fre-FR"). Null by default (takes current locale) |
+| `content` | `eZ\Publish\API\Repository\Values\Content\Content` | Content item the displayed Field belongs to. |
```

Comment:
Same remark: I suppose the change of "object" by "item" is validated with the Repository Team.

Reply:
"Content object" is 5.x, "Content item" is Platform >= 1.x. This change was introduced some two years ago and that's what we're using throughout the doc. Do you see any reason to keep "object" here?

Reply:
> "Content object" is 5.x, "Content item" is Platform >= 1.x. This change was introduced some two years ago and that's what we're using throughout the doc.

Wow, I didn't know about that. I use "Content Object Item". So sometimes this leads to "Content Object Item Version Draft Translation" ;-)
I think there's a lot of inconsistency, so I'm fine with using Content item everywhere.

Reply:
Now we will have to change it clearly and/or t make a Glossary?

Reply:
It is already in the glossary in the user guide: https://doc.ezplatform.com/projects/userguide/en/master/glossary/#content-items, and, more importantly, in our content model overview: https://doc.ezplatform.com/en/1.11/guide/content_model/#content-items-content-types-and-fields
But it is true that a unified glossary is something we'll need to work on in the future.